src/utils/azure_storage.ts

```diff
@@ -254,7 +254,7 @@ export const getBlobAsObject = async <A, O, I>(
           )
         );
       } catch (e) {
-        return E.left(e);
+        return E.left(new Error(`${e}`));
```

Comment:
This is not equivalent because we are generating a new error with message equal to the stringified catched error.
If the error is always an `Error` object we can use the `as` otherwise this must be handled.

Reply:
You are right, according to the `JSON.parse()` documentation this method only throws with a [SyntaxError](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/SyntaxError) which is a subclass of `Error`, as you said we can safely use a cast here IMHO.

Reply:
@BurnedMarshal done in [17eb856](https://github.com/pagopa/io-functions-commons/pull/315/commits/17eb85686a7abeec27e9a8f1b65348673071f2d6)